<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_subwidget</class>
 <widget class="QWidget" name="Form_subwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GPS</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_17">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="3">
       <widget class="QLabel" name="label_Y">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_64">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_dZ">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_Z">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_dX">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="label_61">
        <property name="text">
         <string>RAF09</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_68">
        <property name="text">
         <string>dZ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_65">
        <property name="text">
         <string>dX</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_67">
        <property name="text">
         <string>dY</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_dY">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_62">
        <property name="text">
         <string>Zgps</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_66">
        <property name="text">
         <string>Z mNGF</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_zgps">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_hautperche">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_70">
        <property name="text">
         <string>Rod height</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_zwgs84">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_63">
        <property name="text">
         <string>Zwgs84</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_X">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_69">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLabel" name="label_raf09">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="8">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
